pravega/pravega
# Issue #1878: Upgrade dependencies

**Change log description**
Updates all of our dependencies to their latest published version.

**Purpose of the change**
Fixes #1878 

**What the code does**
Changes our the version numbers in gradle.properties, and makes the required code changes to build. (Auto-generated source has been re-generated.)

**How to verify it**
We will need to re-run all tests (including system and integration tests) to verify there have not been any subtle impacts.
